servicebroker/test/ut/api/api_test.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var should = require('should');
+var API = require(path.join(__dirname, '../../../lib/api/api.js'));
+
+var api = new API();
+
+describe('Unit Test for API', function() {
+
+  it('getCatalog() should return catalog info with JSON format', function() {
+    var data = api.getCatalog();
+    data.should.have.property('services');
```

Comment:
yes, its more important here than in the index test which can do the same check or validate a catalog jsom document is returned.